build.gradle

```diff
@@ -84,6 +85,14 @@ apply from: "${scriptDir}/rat.gradle"
 subprojects {
   // Make sure clean task for rootProject runs last
   clean.finalizedBy rootProject.cleanAll
+
+  apply plugin: "com.diffplug.gradle.spotless"
+  spotless {
+    java {
+//      googleJavaFormat()
```

Comment:
I pushed a commit to remove this line.  Was playing around with Spotless' built-in googleJavaFormat() support (as opposed to bringing in Google Java style via the Eclipse xml), but spotlessCheck would fail even when called immediately after spotlessApply..